pkg/controller/controller.go

```diff
@@ -77,6 +77,25 @@ func RegisterInformers(c *config.Config) {
 				sendEvent(obj, nil, c, kind, config.ErrorEvent)
 			}
 		},
+		UpdateFunc: func(old, new interface{}) {
+			_, err := cache.MetaNamespaceKeyFunc(new)
+			if err != nil {
+				log.Logger.Errorf("Failed to get MetaNamespaceKey from event resource")
+				return
+			}
+			eventObj, ok := new.(*apiV1.Event)
+			if !ok {
+				return
+			}
+
+			// Kind of involved object
+			kind := strings.ToLower(eventObj.InvolvedObject.Kind)
+
+			// Send events update notifs if InvolvedObject is node
+			if kind == "node" {
```

Comment:
@aananthraj 
1. What type of event.Type we get? `warnings/info`
2. If it's `warnings`, does it make sense to send events for all the resources configured with `error` events instead of hardcoding kind here?

Reply:
> If it's warnings, does it make sense to send events for all the resources configured with error events instead of hardcoding kind here?

I guess update events on Kubernetes `Events objects` will mostly be repetitve and hence can become noisy

Reply:
We get two types of events - `normal` and `warnings`. Out of which we are allowing only `warnings` events (https://github.com/infracloudio/botkube/blob/ff21d65c300b008b895fb3e6bf6fda9cb08935c6/pkg/controller/controller.go#L76). Please check if we get lots of redundant `warning` events, if not then we should send events for those

Reply:
resolved this issue. @PrasadG193 kindly check.